AFQ/tasks/data.py

```diff
@@ -280,6 +280,24 @@ def dti_fa(dti_tf):
     return dti_tf.fa
 
 
+@pimms.calc("dti_lt0", "dti_lt1", "dti_lt2", "dti_lt3", "dti_lt4", "dti_lt5")
```

Comment:
Do we want `dki_lt` elements as well? I believe there are 15 elements in DKI. 

And relatedly, we often calculate FA/MD from DKI, when that's doable (i.e., when there are multiple non-zero b-values). Do we want to be able to extract the six DTI elements from DKI?